bindings_node/src/conversations.rs

```diff
@@ -97,6 +98,21 @@ impl From<ListConversationsOptions> for GroupQueryArgs {
   }
 }
 
+#[napi(object)]
+pub struct NodeHMACKey {
```

Comment:
can you make this just `HmacKey` and alias the import as `XmtpHmacKey`? it makes the generated types usable without having to rename them in the SDK.

Reply:
Done